definitions/divorce/json/CaseEvent/CaseEvent-resp-journey-prod.json

```diff
@@ -72,5 +72,20 @@
     "PostConditionState": "AosAwaiting",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/confirm-process-server-service",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "ID": "issueAosFromReissue",
+    "Name": "Issue AOS pack to respondent",
+    "Description": "Issue AOS pack to respondent",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "AwaitingReissue",
+    "PostConditionState": "AosAwaiting",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/bulk-print",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/petition-updated",
```

Comment:
is there logic from this endpoint no longer needed or covered in the aos-pack-issued endpoint?

Reply:
Hi @shaunganley. Yes, the new endpoint has the same logic plus the extra async logic.
You can see it here: https://github.com/hmcts/div-case-orchestration-service/pull/1332